controllers/managersController.js

```diff
@@ -1,16 +1,28 @@
-import Managers from '../models/loginModel';
+import bcrypt from 'bcrypt';
+// import { json } from 'express';
+import Managers from '../models/managersModel';
 
-const managersSignup = async (req, res) => {
+export const managersSignup = async (req, res) => {
   const { email, password } = req.body;
   try {
-    const manager = await Managers.create({
-      email,
-      password,
-    });
-    res.status(201).json({
-      success: true,
-      message: ' manager created',
-      manager,
+    await bcrypt.hash(password, 10, (err, hash) => {
+      if (err) {
+        console.log(err);
+        res.status(500).json({
+          message: 'auth failed',
+        });
+      } else {
+        console.log(hash);
+        const manager = Managers.create({
```

Comment:
please provide all managers details i.e: ​employee name, national id, phone number, date of birth, status​(active, inactive)​ and position​(manager)

Reply:
please check your response on manager signup! 
to fix it you can add an await on Manager.create()

Reply:
thanks, i just corrected this

Reply:
what about the phone number search ??